.github/workflows/_buildpacks-release.yml

```diff
@@ -70,10 +70,12 @@ jobs:
       - name: Rust cache
         uses: Swatinem/rust-cache@v2.6.2
 
+      # the version of `libcnb-cargo` installed here is kept in sync with the version of `libcnb-package`
+      # that the release automation CLI tooling depends on
       - name: Install libcnb-cargo
-        # TODO: Derive this version from the `libcnb-package` version in this repo's `Cargo.toml`,
```

Comment:
The comment here can now be removed too :-)
https://github.com/heroku/languages-github-actions/blob/e9a028c29594d48faf9d9b0123b333c8bcd63cb4/Cargo.toml#L28

Reply:
removed